### PR TITLE
ONS-UK-Total-trade

### PR DESCRIPTION
**Pull request by bailed22**

### CODE REVIEW

File: datasets/ONS-UK-Total-trade/main.py [hunk 1]
```diff
@@ -44,69 +42,45 @@ def date_time (date):
...
-cubes.add_cube(scraper, df.drop_duplicates(), "ons-uk-total-trade")
-cubes.output_all()
-trace.render("spec_v1.html")
-
-
-df.dtypes
+df = df[['Period', 'Country', 'Flow', 'Trade Type', 'Marker', 'Value']]
+
+df['Flow'] = df['Flow'].map(lambda x: 'export'  if 'Exports' in x else 'import')
+
```
Comment (santhosh-thangavel) [inline datasets/ONS-UK-Total-trade/main.py:83]: Updated description needs to be added as per the latest publication. You could make use of landing page and "Index" tab in the spread sheet.

metadata.dataset.description = f"""add the latest description here"""

**Review by santhosh-thangavel — CHANGES_REQUESTED** [review]

File: datasets/ONS-UK-Total-trade/main.py [hunk 2]
```diff
@@ -44,69 +42,45 @@ def date_time (date):
...
-df = trace.combine_and_trace(datasetTitle, "combined_dataframe")
 df.rename(columns={'OBS' : 'Value','DATAMARKER' : 'Marker'}, inplace=True)
-#df['Value'] = df['Value'].astype(float)
-df['Value'] = pd.to_numeric(df.Value, errors='coerce')
-df['Marker'].replace('N/A', 'not-collated', inplace=True)
-trace.Marker("replacing N/A with not-collated")
-df['Flow'] = df['Flow'].map(lambda s: s.lower().strip())
-df["Country"] = df["Country"].map(lambda x: pathify(x).upper())
-df['Trade Type'] = df['Trade Type'].apply(lambda x: 'total' if 'Total Trade' in x else 
+df['Trade Type'] = df['Trade Type'].apply(lambda x: 'all' if 'Total Trade' in x else
```
[the remark this answers is not in the document]
Comment (bailed22) [reply datasets/ONS-UK-Total-trade/main.py:75]: okay and noted

File: datasets/ONS-UK-Total-trade/main.py (hunk 1 above)
Comment (bailed22) [reply datasets/ONS-UK-Total-trade/main.py:82]: Have updated all changes as requested.

**Comment by santhosh-thangavel** [comment]
All looks good. Happy to be merged

**Pull request merged**